dotnet/README.md

```diff
@@ -0,0 +1,32 @@
+# Connect-e Samples .NET
+
+Connect-e Samples .NET provides examples of how to get started integrating with connect-e using [.NET](https://dotnet.microsoft.com/) and [Angular](https://angular.io/).
+
+For more information, please check [Connect-e](https://docs.connect.paymentsense.cloud/ConnectE/GettingStarted) documentation.
+
+## Running
+
+.NET API
+
+This project is using .NET 6.0
+
+1. Change setting file [appsettings.json](./Paymentsense.Connect-e.Api/appsettings.json)
+
+2. Open and run solution: [Paymentsense.Connect-e.sln](./Paymentsense.Connect-e.sln)
```

Comment:
step 2 seems to be specific for visual studio, suppose I don't use such IDE, how can I run this solution?

Reply:
I tried to run following the .net docs, but getting some issues. What am I doing wrong?

<img width="1792" alt="Screenshot 2022-07-20 at 08 03 56" src="https://user-images.githubusercontent.com/514719/179919158-2357b000-bf97-4037-aee4-18a714c4964c.png">

https://code.visualstudio.com/docs/languages/dotnet

Reply:
I added instruction on how to run it in consol, can you try now

Reply:
cool, yeah that worked, but unfortunately I am getting a different error now.

<img width="1792" alt="Screenshot 2022-07-20 at 12 11 29" src="https://user-images.githubusercontent.com/514719/179968563-4419dccb-9653-443b-be21-f53b6386c0f9.png">

Reply:
ok, can you run this cert command and let me know what happens? if you can run project after, I assume you never run dot net core 6 app on your machine before?

Reply:
it's asking for some password for a cert, could you make it simple so no certificate is required? HTTPS is not required for the sample, the idea is to make it as easy as possible for the user/developer to run it. 

about running core 6 on my machine, I am not sure, but probably not.

Reply:
yes, you need to do pull

Reply:
it worked now, thanks mate